package.json

```diff
@@ -14,19 +14,20 @@
   "author": "MasterBorn Sp. z o.o.",
   "license": "MIT",
   "dependencies": {},
-  "devDependencies": {
-    "eslint": "^7.7.0"
-  },
   "peerDependencies": {
+    "@typescript-eslint/eslint-plugin": "^4.8.1",
+    "@typescript-eslint/parser": "^4.8.1",
     "@babel/core": "^7.11.1",
     "@babel/eslint-parser": "^7.11.3",
-    "eslint": "^7.7.0",
+    "eslint": "^7.14.0",
     "eslint-config-airbnb": "^18.2.0",
     "eslint-config-airbnb-base": "^14.2.0",
-    "eslint-plugin-import": "^2.22.0",
-    "eslint-plugin-jsx-a11y": "^6.3.1",
-    "eslint-plugin-react": "^7.20.6",
-    "eslint-plugin-react-hooks": "^4.1.0"
+    "eslint-config-airbnb-typescript": "^12.0.0",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jsx-a11y": "^6.4.1",
+    "eslint-plugin-react": "^7.21.5",
+    "eslint-plugin-react-hooks": "^4.2.0",
```

Comment:
**Q** Is it required? I would be opposed to bumping this up if that's not crucial due to the fact of current usage 🤔